jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/SettingsFixture.java

```diff
@@ -67,18 +67,13 @@ static SettingsDecrypter newSettingsDecrypter(Path settingsSecurityFile) {
       injectedDispatcher.setConfigurationFile(settingsSecurityFile.toAbsolutePath().toString());
       setField(DefaultSecDispatcher.class, injectedDispatcher, "_cipher", injectCypher);
 
-      DefaultSettingsDecrypter settingsDecrypter = new DefaultSettingsDecrypter();
-      setField(
-          DefaultSettingsDecrypter.class,
-          settingsDecrypter,
-          "securityDispatcher",
-          injectedDispatcher);
-      return settingsDecrypter;
+      return new DefaultSettingsDecrypter(injectedDispatcher);
```

Comment:
signature changed, injection via reflection no longer necessary.

Reply:
Any chance `DefaultSecDispatcher` similarly allows setting `injectCypher` without using `setField`?

Reply:
No, I checked that, it still requires some reflective injection :(